chapters/ch04.md

```diff
@@ -1,31 +1,38 @@
 # Scale drives machine learning progress
 
 ->
+# 4. Quy Mô Quyết Định Mô Hình Machine Learning
 
 
 Many of the ideas of deep learning (neural networks) have been around for decades. Why are these ideas taking off now?
 
 ->
+Rất nhiều những ý tưởng của deep learning (mạng neural) đã xuất hiện từ hàng thập kỷ trước.  Vậy tại sao tận bây giờ chúng mới cất cánh, mới bùng nổ như vậy?  
 
 
 Two of the biggest drivers of recent progress have been:
 
 ->
+Hai nguyên nhân mang tính quyết định tới những phát triển gần đây của deep learning là
 
 
 * **Data availability.​** People are now spending more time on digital devices (laptops, mobile devices). Their digital activities generate huge amounts of data that we can feed to our learning algorithms.
 
 ->
+**Dữ liệu sẵn có**. Ngày nay,  mọi người dành nhiều thời gian hơn bên những thiết bị số như máy tính xách tay, thiết bị di động, .v.v. Chính những thiết bị số này là nguồn tạo ra lượng dữ liệu cực lớn - dữ liệu vẫn dùng cho những thuật toán học máy ngày nay.  
```

Comment:
* [ ] Hai dấu cách sau dấu phẩy.
* [ ] "... lượng dữ liệu cực lớn cho các thuật toán học.". Không có chữ 'máy' nào ở đây. Nếu có thì mình cần giữ nguyên cụm 'machine learning.